src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/models/Discrepancy.java

```diff
@@ -1,31 +1,23 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.services.reports.models;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
-
 public class Discrepancy {
-    @JsonProperty("zip_file_name")
     public final String zipFileName;
-
-    @JsonProperty("container")
     public final String container;
-
-    @JsonProperty("type")
     public final String type;
-
-    @JsonProperty("stated")
     public final String stated;
-
-    @JsonProperty("actual")
     public final String actual;
```

Comment:
can you explain?

Reply:
There is a new class DiscrepancyItem for Json presentation

Reply:
looks identical, but ok